example/tests/test_views.py

```diff
@@ -124,6 +124,29 @@ def test_relationship_view_errors_format(self):
         assert "data" not in result
         assert "errors" in result
 
+    def test_relationship_view_errors_has_correct_pointers(self):
```

Comment:
`RelationshipView` is a class in `views.py` but that is not what you are testing here. This should rather be added to `test_errors.py`.

Reply:
Sure!